package.json

```diff
@@ -4,8 +4,8 @@
   "private": "true",
   "scripts": {
     "storybook": "start-storybook -p 6006",
-    "initialize": "npm install && npm run clean && lerna bootstrap --hoist",
-    "clean": "lerna clean --yes",
+    "initialize": "npm run clean && npx lerna bootstrap --hoist",
```

Comment:
In this case you won't run `npm i` since `npm i` is dumb and does a recursive install of the packages in each sub-package within this monorepo - which sucks. However, we need `lerna` inside our project which is specified inside the root `package.json` to do the "smart" package management through `lerna bootstrap --hoist`. 

So in this case i'm using `npx` which downloads the lerna application for us inside this single run without needing to rely on `package.json`. lerna then technically installs itself inside the project using `lerna bootstrap --hoist`.

Reply:
Ah, I see...